### ISSUE TITLE
Pre-trained PWNet model

### ISSUE THREAD

**Issue by alexlopezcifuentes** [mindazhao/pix-pix-warping-video-stabilization#1]
Do you have plans of releasing the pre-trained model of PWNet for testing?

**Comment by mindazhao** [comment]
Yes, I have the plan. However, as the impact of COVID-19, I can not go to the lab and get the pre-trained model. I will release it when I am allowed back.
